### ISSUE TITLE
replication performance  poor

### ISSUE THREAD

**Issue by sheldoncooper5** [zqyq/Multi-view-People-Detection-in-Large-Scenes-via-Supervised-View-wise-Contribution-Weighting#4]
Training replication using the citystreet dataset is poor, especially on 5 evaluation metrics (MODA MODP precision recall F1) far less than the performance in the paper

**Comment by sheldoncooper5** [comment]
[log.txt.txt](https://github.com/user-attachments/files/18274946/log.txt.txt)

**Comment by BruceYounggit** [comment]
Hi, after reading your log.txt, I speculate that the training loss does not completely converge, maybe you can train the model longer. And, because this model is trained through three-stage training, you must assure that each stage has a good performance, for instance, the second stage should give out a relatively good single-view prediction results ( you can check it by the corresponding  visulization results). Based on the previous pre-trained model, the final model will have a nice initialization so that it can achieve the global minimum point of the loss curve with the best performance.

**Comment by sheldoncooper5** [comment]
grateful for your reply! I tried to train the every stages (including single-view prediction) until every loss keep stable, but the performance is still poor. Can you share the pertained model checkpoint in citystreet dataset? I want to ensure whether the dataset I downloaded makes this problems( I think maybe the citystreet dataset need some data preprocessing？ or maybe I'm using an incorrect data label file).  because around 40% of moda are really confusing

**Comment by sheldoncooper5** [comment]
Also I used the CVCS.pth that you provided to  test the model performance and the metrics I got were equally low in moda and F1score, moda was 10.8% and F1score was 19.5%, I'm attaching the log file, can you answer why this is?
[log.txt.txt](https://github.com/user-attachments/files/18285619/log.txt.txt)

**Comment by BruceYounggit** [comment]
You can experiment with different `cls_thres` values to achieve better performance. For the CityStreet test, I used `cls_thres = 0.5`. Regarding the pretrained model checkpoint, it was trained two years ago and has different keys in the `model_dict` due to changes in the network settings, which causes a slight drop in performance. As for the CVCS.pth, it also can be improved by taking suitable 'cls_thres' which I explained under the downloading path.
